timecog/timecog.py

```diff
@@ -748,14 +761,19 @@ def ydhm(seconds):
     return " ".join(ydhm) or "<1 minute"
 
 
-def format_rm_time(rmtime, input, D_TZ):
-    return "'{}' on {} {} ({} from now)".format(
-        input,
-        D_TZ.fromutc(rmtime).strftime(DT_FORMAT),
-        get_tz_name(D_TZ, rmtime),
-        ydhm((rmtime - datetime.utcnow()).total_seconds() + 2)
-    )
-
+def format_rm_time(rmtime, input, D_TZ, show_tz):
+    if show_tz:
```

Comment:
I would do the least-complicated case first

then because you *returned* you don't need to put the more-complicated case in an `else` block, you can just de-indent. Yay!

This goes in conflict a bit to "always do if <positive condition>" so you have to balance those two conventions but if you can do a less-complicated case and then de-indent your more complicated case, and the logic of the "negative condition" is really straightforward I'll always go for it yay